### DISCUSSION TITLE
Resume interupted script

### DISCUSSION THREAD

**Discussion by AleytiaFairlight** [tteck/Proxmox#3504] [Help Needed]
Is it possible to resume a script that has been interrupted? while installing vault warden it wasnt able to connect to a repo due to a timeout and the script stopped. is there are way to contiue?

I don't have the error when installing vault warden cause I accidentaly closed the console but here is the error from trying to install paparless-ngl

```
[ERROR] in line 53: exit code 0: while executing command $STD apt-get install -y --no-install-recommends unpaper ghostscript icc-profiles-free qpdf liblept5 libxml2 pngquant zlib1g tesseract-ocr tesseract-ocr-eng
```

Update here is the error wgen installing vaultwarden which is also related to internet timeout.

```
[ERROR] in line 33: exit code 0: while executing command WEBVAULT=$(curl -s https://api.github.com/repos/dani-garcia/bw_web_builds/releases/latest | grep "tag_name" | awk '{print substr($2, 2, length($2)-3) }')
```

**Comment by CrazyWolf13** [comment]
No I don't think so, I suggest the easiest is just to remove the container from the web-ui and then just start the installer again

**Reply by AleytiaFairlight** [reply]
err that's not going to work for me as the connection will always time out at some point as my internet plan has a connection limit and with 15 devices connected that connection limit is met more often than not.

**Reply by CrazyWolf13** [reply]
Well to install something a internet connection is kinda required lol 

You may be able to skip the updates, but you'd still have to make sure the dependencies are installed.

I don't think tteck will modify his whole set of scripts just to allow resuming, as for most users this is 99% useless.

I suggest you download the script with `wget` and then manually edit the lines, then you can start it again.

**Reply by AleytiaFairlight** [reply]
I get that an internet connection is required. hmm I suppose what I'm asking is if it's possible to have a retry when the connection times out so it can reconnect.

**Reply by CrazyWolf13** [reply]
apt usually tries quite a long time to reconnect, you can try setting debug mode at I think the second last prompt, then you can see when timeouts occur.

**Reply by AleytiaFairlight** [reply]
I have venbose active which is why I know which line fails. I don't think the whole script needs to be modified for adding a retry function.

**Reply by Mellowlynx** [reply]
Have a friend with better internet and also has Proxmox?
Ask him to create the LXC on that machine and use a backup to restore it on yours.
But I think the obvious way to fix, it to get better internet and stop accepting bull s*it limitations.
